lib/generators/draft/resource/templates/views/index.html.erb

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Is this additional line in here on purpose or did it sneak in?

Reply:
No reason for that to be there. I tried to pop out all of my extra spaces, must have missed that one, let me pop that one out.

Reply:
I didn't know about `rubocop -a` when I started this gem so it would do a ton of changes right now; we can save that for a future card. I was mostly just curious.

Reply:
I ran `rubocop -a` and there seemed to be a few errors with rubocop in the gem.  I didn't spend much time looking at how to fix it, but yeah maybe a future card